docs/html/conf.py

```diff
@@ -12,22 +12,30 @@
 sys.path.insert(0, docs_dir)
 
 # -- General configuration ------------------------------------------------------------
-
 extensions = [
-    # first-party extensions
-    "sphinx.ext.autodoc",
-    "sphinx.ext.todo",
-    "sphinx.ext.extlinks",
-    "sphinx.ext.intersphinx",
-    # our extensions
+    # extensions common to 'man' and 'html' builds
     "pip_sphinxext",
-    # third-party extensions
-    "myst_parser",
-    "sphinx_copybutton",
-    "sphinx_inline_tabs",
-    "sphinxcontrib.towncrier",
 ]
 
+# 'tags' is a special object handled by sphinx
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-tags
+if "html" in tags or "READTHEDOCS" in os.environ:  # type: ignore[name-defined] # noqa: F821
```

Comment:
FWIW, I think the idiomatic check is `tags.has('html')`.

Reply:
Also note that it's not just `html` but `htmldir` too.

Reply:
> FWIW, I think the idiomatic check is `tags.has('html')`.

I was just following the docs on that one https://www.sphinx-doc.org/en/master/usage/configuration.html#project-tags (look to be a recent update <1 year ago https://github.com/sphinx-doc/sphinx/pull/12490)

Reply:
Ah, makes sense.